ceph-dev-build/build/build_mingw

```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
 set -ex
 
-# We need Ubuntu Focal to cross-compile Ceph for Windows.
-# Previous versions provide broken Mingw packages.
+# We need Ubuntu jammy to cross-compile Ceph for Windows.
```

Comment:
Nit: I think Jammy is to be capitalized just like Focal used to be ;)

Reply:
We'll eventually have to bump it again, using the same casing makes it easier to find/replace. If you prefer, I can update it.

Reply:
It looks like you updated both `setup_mingw` files but neither of `build_mingw` files.  This is really a nit and not blocking by any means but I think it's better to be consistent.  `git grep -i` works pretty well ;)

Reply:
Oh right, my bad. I'll send a new patch.